packages/bpk-component-navigation-bar/src/BpkNavigationBarIconButton.tsx

```diff
@@ -39,17 +39,18 @@ export type Props = {
 };
 
 const BpkNavigationBarIconButton = ({ barStyle = BAR_STYLES.default, className, icon, ...rest }: Props) => (
-  <BpkIconButton
-    customIcon={icon}
-    // TODO: className to be removed
-    // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-    className={getClassName(
-      'bpk-navigation-bar-icon-button',
-      `bpk-navigation-bar-icon-button--${barStyle}`,
-      className
-    )}
-    {...rest}
-  />
+  <span className={className}>
+    <BpkCloseButton
+      customIcon={icon}
+      // TODO: className to be removed
+      // eslint-disable-next-line @skyscanner/rules/forbid-component-props
+      className={getClassName(
```

Comment:
Just to be clear, we're leaving this className and the comment for now, right?

Reply:
Yeah the BpkCloseButton is out of scope for this ticket. Otherwise would have to update everything else that uses it.